### PR TITLE
Fix date in CHANGELOG

### PR DESCRIPTION
**Pull request by OndraM**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| Documentation   | -
| License         | MIT

#### What's in this PR?

Just a fix of typo in date in chagelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## 1.6.0 - 2017-01-29
+## 1.6.0 - 2018-01-29
```
Comment (GrahamCampbell) [inline CHANGELOG.md:3]: I think this is meant to say "Upcoming"?

Comment (OndraM) [reply CHANGELOG.md:3]: This line was added on 2018-01-29 (ie. previous year).

Comment (GrahamCampbell) [reply CHANGELOG.md:3]: 1.6.0 has not been released yet

**Comment by sagikazarmark** [comment]
Looks like there was some confusion here...fixed in https://github.com/php-http/cache-plugin/commit/a250aa2f14a49884c69405e99877cc0947c8d54a

Thanks!

**Pull request closed without merging**